mockall/tests/mock_generic_method_with_unknown_size_type_bound.rs

```diff
@@ -0,0 +1,76 @@
+// vim: tw=80
```

Comment:
It looks like you copied this test program from mock_generic_method_with_where_clause.rs.  In that case, it really was necessary to have the four different methods, because different code gets emitted in each case.  But here, having the four separate methods doesn't give you any better coverage.  In fact, even testing mock and automock separately doesn't improve the code coverage of this feature.  I think you can eliminate this file altogether.